bin/hdfcoinc/pycbc_multiifo_coinc_statmap_inj

```diff
@@ -0,0 +1,157 @@
+
+if args.verbose:
+    log_level = logging.INFO
+    logging.basicConfig(format='%(asctime)s : %(message)s', level=log_level)
+
+logging.info("Loading coinc zerolag triggers")
+window = args.cluster_window
+zdata = pycbc.io.MultiifoStatmapData(files=args.zero_lag_coincs, ifos = args.ifos)
+zdata = zdata.cluster(window)
+
+logging.info("Loading coinc full inj triggers")
+fidata = pycbc.io.MultiifoStatmapData(files=args.mixed_coincs_full_inj, ifos = args.ifos).cluster(window)
+
+logging.info("Loading coinc inj full triggers")
+ifdata = pycbc.io.MultiifoStatmapData(files=args.mixed_coincs_inj_full, ifos = args.ifos).cluster(window)
+
+f = h5py.File(args.output_file, "w")
+
+f.attrs['num_of_ifos'] = zdata.attrs['num_of_ifos']
+f.attrs['pivot'] = zdata.attrs['pivot']
+f.attrs['fixed'] = zdata.attrs['fixed']
+f.attrs['timeslide_interval'] = zdata.attrs['timeslide_interval']
+
+# Copy over the segment for coincs and singles
+for key in zdata.seg.keys():
+    f['segments/%s/start' % key] = zdata.seg[key]['start'][:]
+    f['segments/%s/end' % key] = zdata.seg[key]['end'][:]
+
+logging.info('writing zero lag triggers')
+if len(zdata) > 0:
+    for key in zdata.data:
+        f['foreground/%s' % key] = zdata.data[key]
+else:
+    for key in zdata.data:
+        f['foreground/%s' % key] = numpy.array([], dtype=zdata.data[key].dtype)
+
+logging.info('calculating statistics excluding zerolag')
+fb = h5py.File(args.full_data_background, "r")
+
+# Allow all time to contribute to injection calculated IFARs, as for zerolag
+# This prevents loud injections from being assigned to a bin where they have
+# bad chisq if that bin has slightly larger exclusive bg livetime due to
+# different censoring
+background_time = float(fb.attrs['background_time'])
+coinc_time = float(fb.attrs['foreground_time'])
+back_stat = fb['background_exc/stat'][:]
+dec_fac = fb['background_exc/decimation_factor'][:]
+
+f.attrs['background_time_exc'] = background_time
+f.attrs['foreground_time_exc'] = coinc_time
+f.attrs['background_time'] = background_time
+f.attrs['foreground_time'] = coinc_time
+
+if len(zdata) > 0:
+    fnlouder_exc = coinc.calculate_n_louder(back_stat, zdata.stat, dec_fac, skip_background=True)
+    ifar_exc = background_time / (fnlouder_exc + 1)
+    fap_exc = 1 - numpy.exp(- coinc_time / ifar_exc)
+    f['foreground/ifar_exc'] = conv.sec_to_year(ifar_exc)
+    f['foreground/fap_exc'] = fap_exc
+
+    logging.info('calculating injection backgrounds')
+    fore_time = {}
+    for ifo in args.ifos:
+        fore_time[ifo] = zdata.data['%s/time' % ifo]
+    ftimes = 0
+    for ifo in args.ifos:
+        ftimes += fore_time[ifo] / len(args.ifos)
+    start, end = ftimes - args.veto_window, ftimes + args.veto_window
+
+    fnlouder = numpy.zeros(len(ftimes), dtype=numpy.float32)
+    ifar = numpy.zeros(len(ftimes), dtype=numpy.float32)
+    fap = numpy.zeros(len(ftimes), dtype=numpy.float32)
+
+    # We are relying on the injection data set to be the first one,
```

Comment:
I don't know what these comments mean - have asked @ahnitz to think about if they are meaningful or needed

Reply:
@tdent It means that the injection file and the full data file must be in a particular order. This is also indicated by the naming convention (i.e. if vs fi) and the corresponding argument names. Otherwise, I'm not sure the sorting will work out corectly.

Reply:
So anything decided about this?

Reply:
I think I know what this is referring to .. so I would change this comment to something like "we expect the injfull file to contain injection data as pivot and fullinj to contain full data as pivot" 
I don't think anything should depend on the _ordering_ of input files to multiifo coinc findtrigs !